src/seals/util.py

```diff
@@ -118,3 +118,39 @@ def sample_distribution(
 def one_hot_encoding(pos: int, size: int) -> np.ndarray:
     """Returns a 1-D hot encoding of a given position and size."""
     return np.eye(size)[pos]
+
+
+def grid_transition_fn(
```

Comment:
I've commented on this before in https://github.com/HumanCompatibleAI/seals/pull/23 -- make sure you address those comments and merge the previous PR before this one.